### PR TITLE
feat(operator) Add a function to calculate the usd value for staker

### PR DESCRIPTION
**Pull request by TimmyExogenous**
## Description

This PR adds a function to calculate the USD value for stakers, which might be used for a simple version of reward distribution. 

## Summary by CodeRabbit

- **New Features**
	- Introduced new methods for managing USD values for operators, including deletion capabilities for AVS USD values.
	- Added methods for retrieving specified asset information by operator in the AssetsKeeper interface.
	- Enhanced delegation management with a new iteration method for the DelegationKeeper interface.

- **Bug Fixes**
	- Improved error handling across several methods to prevent potential issues with asset price retrieval.

- **Refactor**
	- Streamlined error registrations and removed several redundant error codes, enhancing clarity in error handling.
	- Renamed methods and store prefixes related to operator delegation for better clarity and concept alignment.

- **Style**
	- Minor formatting adjustments made for better code readability, including whitespace and import statement management.

### CODE REVIEW

File: x/operator/keeper/usd_value.go [hunk 1]
```diff
@@ -240,3 +262,189 @@ func (k *Keeper) GetOperatorAssetValue(ctx sdk.Context, operator sdk.AccAddress,
...
+	} else {
+		optedUSDValues, err = k.GetOperatorOptedUSDValue(ctx, avsAddrString, operator.String())
+		if err != nil {
+			return operatortypes.OperatorOptedUSDValue{}, err
+		}
+	}
+	return optedUSDValues, nil
+}
+
+func (k *Keeper) CalculateUSDValueForStaker(ctx sdk.Context, stakerID, avsAddr string, operator sdk.AccAddress) (sdkmath.LegacyDec, error) {
```
Comment (mikebraver) [inline x/operator/keeper/usd_value.go:390]: Can you also add voting power calculation for staker?

**Review by mikebraver — APPROVED** [review]

File: x/operator/keeper/usd_value.go [hunk 2]
```diff
@@ -1,9 +1,13 @@
 package keeper
 
 import (
+	"errors"
 	"fmt"
 
 	assetstype "github.com/ExocoreNetwork/exocore/x/assets/types"
+	delegationkeeper "github.com/ExocoreNetwork/exocore/x/delegation/keeper"
+	delegationtype "github.com/ExocoreNetwork/exocore/x/delegation/types"
+	oracletypes "github.com/ExocoreNetwork/exocore/x/oracle/types"
```
Comment (bwhour) [inline x/operator/keeper/usd_value.go:10]: oracletype
keep the uniform style is better.

**Review by MaxMustermann2 — APPROVED** [review]

File: x/operator/keeper/usd_value.go (hunk 2 above)
Comment (TimmyExogenous) [reply x/operator/keeper/usd_value.go:10]: Thanks

**Review by leonz789 — APPROVED** [review]

File: x/operator/keeper/usd_value.go (hunk 2 above)
Comment (TimmyExogenous) [reply x/operator/keeper/usd_value.go:10]: This will be fixed in the next PR.

**Pull request merged**